src/components/learner-credit-management/data/utils.js

```diff
@@ -664,3 +622,77 @@ export const getNormalizedEnrollByDate = ({ enrollBy }) => {
   }
   return enrollBy;
 };
+
+/**
+ * Filters assignable course runs based on the following criteria:
+ *  - If hasEnrollBy, we return assignments with enroll before the soonest of the two date: The subsidy expiration
+ *    date - refund threshold OR today offset by the 90-day allocation threshold for an assignment denoted as
+ *    isEligibleForEnrollment
```

Comment:
Nit: clarify that "the purpose of making run visibility sensitive to the subsidy expiration date is to incentivize renewal, and the purpose of making run visibility sensitive to today's date is to ..." (fill in the ellipsis).

Reply:
Updated the docstring language to be specific of the intentions 👍🏽